torch/csrc/jit/ir/ir.cpp

```diff
@@ -1201,7 +1201,7 @@ void Node::destroy() {
     eraseBlock(blocks().size() - 1);
   }
   removeAllInputs();
-  if (inBlockList()) {
```

Comment:
It's Block, not Black here.

Reply:
How did this compile lol

Reply:
Oh it changed everything at once, makes sense.

This change happened because I first changed `blacklist` -> `blocklist` and then `blocklist` -> `denylist`.

Reply:
Reverted.